css-four-column-layout/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS 4 Column Layout</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="row center">
+    <h2>How It Works</h2>
+  </div>
+  <div class="row center">
+    <div class="col-10">
+      <!-----EMPTY-->
+    </div>
```

Comment:
You shouldn't ever have empty columns just to take up space. You can get this behavior by just adding a class with `justify-content: center` to the `row` class parent element.

Reply:
ah okay! good to know moving forward, empty columns removed. thanks Cody!